### ISSUE TITLE
Add copyright notice to the license header in each file

### ISSUE THREAD

**Issue by davidhedlund** [nohamelin/simple-locale-switcher#15]
There are license notices in each file but no copyright notice.

I will approve https://directory.fsf.org/wiki/IceCat/Simple_Locale_Switcher if you fix this.

Thank you for providing a great free software.

**Comment by davidhedlund** [comment]
[mylabel:copyright](https://github.com/search?utf8=%E2%9C%93&q=commenter%3Adavidhedlund+%22mylabel%3Acopyright%22+in%3Acomments&type=Issues&ref=searchresults)

**Comment by nohamelin** [comment]
A copyright notice too? I can do it, I want to reread some related FSF docs first.

**Comment by davidhedlund** [comment]
[mylabel:AMO](https://github.com/search?utf8=%E2%9C%93&q=commenter%3Adavidhedlund+%22mylabel%3AAMO%22+in%3Acomments&type=Issues&ref=searchresults)

**Comment by nohamelin** [comment]
I have barely looked over it, but I'm guessing I need to add each team of translators to the copyright notices in the respective locale files too. Babelzilla will not help me to automatize any of it.

**Comment by davidhedlund** [comment]
@nohamelin You don't have to translate the license notices to other languages. English is fine for all of them.

**Comment by nohamelin** [comment]
@davidhedlund No, I was referring about the names of each translator being added to the respective files containing the translations done by him.

**Comment by davidhedlund** [comment]
@nohamelin Now I understand, yes you should do that of course.
